src/components/common/InfoTags.tsx

```diff
@@ -0,0 +1,54 @@
+import { Stack, Text } from "@chakra-ui/react"
```

Comment:
Every change related to this file should be reverted to it's state on main

Reply:
My bad, I'll completely delete this file, we're using the one which is located in `components/community/levels/components/InfoTags.tsx`.